commit 987dbb02f0bf483898b3a2d7b98411da111012c1
Author: Alexander Goscinski <alex.goscinski@posteo.de>
Date:   Fri Jul 26 15:57:49 2024 +0200

    fix tests

diff --git a/tests/conftest.py b/tests/conftest.py
index b289222..81c5d0c 100644
--- a/tests/conftest.py
+++ b/tests/conftest.py
@@ -89,9 +89,10 @@ class ComputerFirecrestConfig:
     token_uri: str
     client_id: str
     client_secret: str
-    machine: str
+    compute_resource: str
     temp_directory: str
     workdir: str
+    api_version: str
     small_file_size_mb: float = 1.0

 class RequestTelemetry:
@@ -154,7 +155,7 @@ def firecrest_config(
                 config.client_id, config.client_secret, config.token_uri
             ),
         )
-        client.mkdir(config.machine, config.scratch_path, p=True)
+        client.mkdir(config.compute_resource, config.scratch_path, p=True)

         if record_requests:
             telemetry = RequestTelemetry()
@@ -171,7 +172,7 @@ def firecrest_config(
         # because they use `rm -r`:
         # https://github.com/eth-cscs/firecrest/blob/7f02d11b224e4faee7f4a3b35211acb9c1cc2c6a/src/utilities/utilities.py#L347
         if not no_clean:
-            client.simple_delete(config.machine, config.scratch_path)
+            client.simple_delete(config.compute_resource, config.scratch_path)

         if telemetry is not None:
             test_name = request.node.name
@@ -201,6 +202,7 @@ def firecrest_config(
             client_id="CLIENT_ID",
             client_secret=str(_secret_path),
             compute_resource="MACHINE_NAME",
+            workdir=str(workdir),
             small_file_size_mb=1.0,
             temp_directory=str(_temp_directory),
             api_version="2",
